src/main.rs

```diff
@@ -50,23 +51,29 @@ struct Args {
 /// The `main()` method. Will loop forever to serve HTTP requests.
 fn main() {
     if let Err(err) = env_logger::init() {
-        panic!(format!("failed to initialize logger: {}", err.description()));
+        println!("Failed to initialize logger: {}", err.description());
+        exit(1);
```

Comment:
That is pretty minor, but I feel like others might prefer to have that exit code change in a separate PR. I think it does not matter much, it is a good improvement.